messages/rule-crashed.txt

```diff
@@ -0,0 +1,21 @@
+ESLint encountered an uncaught exception from rule "<%- ruleName %>".
+
+<% if (pluginName) { %>
+The rule in question comes from the plugin "eslint-plugin-<%- pluginName %>". You should open an issue on that plugin's issue tracker. Be sure to include your configuration, file source code, and the error message and stack trace.
```

Comment:
We should either show the plugin's version below next to the ESLint version (if that's possible) or include it in the list of info here.